hazel/third_party/cabal2bazel/bzl/cabal_package.bzl

```diff
@@ -425,6 +429,12 @@ def _get_build_attrs(
 
     elibs_targets = _get_extra_libs(build_info.extraLibs, extra_libs)
 
+    # Create per-package unique import of core dependency.
+    rts_import_name = "__hazel_{}_rts".format(name)
+    haskell_toolchain_library(
```

Comment:
Same here. That should already exist in `@haskell_rts//:rts`.

Reply:
Could creating these per-package targets be the reason for the large `(LD_)LIBRARY_PATH`. IIUC each transitive Hackage dependency will add another instance of these libraries to the dependencies.